src/backend/src/modules/configuration/configuration.service.ts

```diff
@@ -0,0 +1,18 @@
+import { HttpException, HttpStatus, Injectable } from '@nestjs/common';
+import * as fs from 'fs';
+@Injectable()
+export class ConfigurationService {
+  getFeatures() {
+    try {
+      const filePath = process.env.CONFIG_FILE_PATH;
+      const configContents = fs.readFileSync(filePath, 'utf8');
+      return JSON.parse(configContents);
+    } catch (error) {
+      console.log(`Read Feature Flag File ERROR: ${error}`);
+      throw new HttpException(
```

Comment:
I think `throw new InternalServerErrorException('Cannot find config file')` is more idiomatic